test_cli.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+lsof -P | grep ':4000' | awk '{print $2}' | xargs kill -9
```

Comment:
This seem a little dangerous... What if there is already something running in port 4000 that is not our test server? I prefer being more conservative. What do you think of creating a command just for stopping the test server that the user has to call manually?

Also, `lsof -P | grep ':4000'` is not very performatic. `lsof -Pi :4000` does the same faster.